apis/r/R/TileDBGroup.R

```diff
@@ -64,6 +64,23 @@ TileDBGroup <- R6::R6Class(
       )
       # TODO: Avoid closing/re-opening the group to update the cache
       self$close()
+
+      # We manually add the new member to member_cache in order to preserve the
+      # original URI. Otherwise TileDB Cloud creation URIs are retrieved from
+      # using tiledb_group_member() in the form tiledb://namespace/uuid. In this
```

Comment:
I would say a "creation URI" would be of the form `tiledb://namespace/s3://bucketname/path`, whereas a "post-creation URI" would be of the for you indicate here, namely, `tiledb://namespace/uuid`.

I don't think "creation URI" or "post-creation URI" are particularly standard terms, but, I don't think `tiledb://namespace/uuid` is something we should be calling a "creation URI".

I'd also at a clarifying bit to your comment, namely, _why_ it's not possible to append new children: please point out explicitly on the comment that this is where we _do_ need URIs of the form `tiledb://namespace/s3://bucketname/path`.